rustyms/src/shared/glycan.rs

```diff
@@ -172,7 +210,16 @@ impl MonoSaccharide {
             index += line[index..].ignore(&["-"]);
         }
         // Another optional isomeric state
-        index += line[index..].ignore(&["d-", "l-", "?-"]);
+        if line[index..].starts_with("d-") {
+            configuration = Some(Configuration::D);
```

Comment:
Heptose and up may actually have two stereoconfigurations, though these are often concealed behind "common names" like neuraminic acids and the like.

NeuGc in GlycoCT is rendered:
```
1b: a-dgro-dgal-NON-2:6|1:a|2:keto|3:d
2s: n-glycolyl
```

Reply:
Yes I saw their existence, but there is no glycan in de current database that has two of these configuration statements on one monosaccharide, so I am not entirely sure how that would look and as such did not want to burn my hands on this issue.

Reply:
I build initial support for storing this info, it is not very properly checked (so it is possible to set DD for a pentose for example) but also not parsed from short iupac, as I did not find any examples.